<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DRO Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Reference Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="refTypCombo">
       <item>
        <property name="text">
         <string>Absolute (G53)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Relative (G5x)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Distance To Go</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Machine Axis</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="axisCombo">
       <item>
        <property name="text">
         <string>X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Z</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>U</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>V</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>W</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Imperial Format (G20)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="inFmtEntry">
       <property name="placeholderText">
        <string>%9.4f</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Metric Format (G21)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="mmFmtEntry">
       <property name="placeholderText">
        <string>%10.3f</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
